sensing/autoware_pointcloud_preprocessor/src/downsample_filter/approximate_downsample_filter_node.cpp

```diff
@@ -64,9 +64,9 @@ ApproximateDownsampleFilterComponent::ApproximateDownsampleFilterComponent(
 : Filter("ApproximateDownsampleFilter", options)
 {
   {
-    voxel_size_x_ = static_cast<double>(declare_parameter("voxel_size_x", 0.3));
-    voxel_size_y_ = static_cast<double>(declare_parameter("voxel_size_y", 0.3));
-    voxel_size_z_ = static_cast<double>(declare_parameter("voxel_size_z", 0.1));
+    voxel_size_x_ = declare_parameter<double>("voxel_size_x");
```

Comment:
nit: the reasons these variables (applicable in all other PRs) are double is that in the past float was not accepted as a parameter (if my memory serves me right). When passed to the PCL structure it is actually cast into a float. This would be an excellent time to makes things "right"

Reply:
Thanks! fixed them in https://github.com/autowarefoundation/autoware.universe/pull/8480/commits/e1525201755674b2f81d3b049883aaacac86a835